docs/user-guide/timeouts/timeouts-main.md

```diff
@@ -0,0 +1,331 @@
+# Introduction
+
+In the following, we will give a basic classification of timeouts, explain some
+candidate causes for timeouts, and show ways to sometimes prevent them. We give
+a glimpse into the more theoretical background of timeouts in program
+verification in [this section](timeouts-theory.md)
+
+We classify Certora Prover timeouts as follows:
+1.  Timeouts that happen before SMT solvers are running 
+2.  Timeouts where the SMT queries in sum lead to a global timeout
+3.  Timeouts where a single SMT query could not be solved 
+
+Types 1. and 2. are signified by a hard stop of the Prover. That means the
+Prover ran until the global timeout (set via `--globalTimeout`, typically 2
```

Comment:
I think the user cannot set timeout higher than 2 hours via `--globalTimeout` (I think there is also ~2h hard limit in our containers or something like that). Though, I am not sure. Anyhow, my point is that this suggests the user can increase the `2h` timelimit via the flag (and I am not sure whether it is the case)